ldap/servers/slapd/slapi-plugin.h

```diff
@@ -8084,7 +8084,55 @@ typedef void (*slapi_eq_fn_t)(time_t when, void *arg);
  *
  * \param fn The function to call when the event is triggered.
  * \param arg An argument to pass to the called function.
- * \param when The time that the function should be called.
+ * \param when The time that the function should be called(MONOTONIC clock).
+ *
+ * \return slapi_eq_context
+ */
+Slapi_Eq_Context slapi_eq_once_rel(slapi_eq_fn_t fn, void *arg, time_t when);
```

Comment:
` Slapi_Eq_Context slapi_eq_once_rel(slapi_eq_fn_t fn, void *arg, time_t when) __attribute__((deprecated));`

Is what I meant with adding deprecated attributes so that it causes gcc to throw a warning to anyone consuming this api.

Reply:
why would we deprecate the new function? :-)

Reply:
Clearly I'm low on caffeine :) 

If the attribute deprecated is on the *old* function then I'm happy :)

Reply:
Changes made please review @Firstyear